zokrates_js/package.json

```diff
@@ -31,8 +39,8 @@
     "build:bundler:dev": "rimraf pkg && npm run wasm-pack -- --target bundler --dev && npm run clean-pkg",
     "build:node": "rimraf node/pkg && npm run wasm-pack -- --target nodejs -d node/pkg --release && npm run clean-node-pkg",
     "build:node:dev": "rimraf node/pkg && npm run wasm-pack -- --target nodejs -d node/pkg --dev && npm run clean-node-pkg",
-    "clean-pkg": "rimraf pkg/README.md",
-    "clean-node-pkg": "rimraf node/pkg/README.md",
+    "clean-pkg": "rimraf pkg/README.md pkg/.gitignore",
+    "clean-node-pkg": "rimraf node/pkg/README.md node/pkg/.gitignore",
```

Comment:
No blocking but just making sure this is intended

Reply:
It is intended yes, it seems like these pkg folders were not being included in the tarball because of this .gitignore files